creator-node/src/components/replicaSet/replicaSetController.js

```diff
@@ -81,54 +79,77 @@ const syncRouteController = async (req, res) => {
     )
   }
 
+  const wallet = walletPublicKeys[0]
```

Comment:
is this the change from wallets array to single wallet? is the reason it's still accepting an array for backwards compat?

Reply:
yea, it's not worth waiting on this for a libs bump. gonna leave alone for now